paparazzi/paparazzi/src/test/java/app/cash/paparazzi/PaparazziTest.kt

```diff
@@ -115,6 +116,65 @@ class PaparazziTest {
     )
   }
 
+  @Test
+  fun animationCallbacksForStaticSnapshots() {
+    val log = mutableListOf<String>()
+
+    val view = object : TextView(paparazzi.context) {
+      override fun onDraw(canvas: Canvas) {
+        log += "onDraw text=$text"
+      }
+    }
+
+    val animator = ValueAnimator.ofInt(200, 300)
+    animator.addUpdateListener {
+      view.text = it.animatedFraction.toString()
+    }
+    animator.addListener(object : AnimatorListenerAdapter() {
+      override fun onAnimationStart(animation: Animator?, isReverse: Boolean) {
+        log += "onAnimationStart uptimeMillis=$time"
+      }
+
+      override fun onAnimationEnd(animator: Animator) {
+        log += "onAnimationEnd uptimeMillis=$time"
+      }
+    })
+
+    animator.startDelay = 2_000L
+    animator.duration = 1_000L
+    animator.interpolator = LinearInterpolator()
+    assertThat(AnimationHandler.getAnimationCount()).isEqualTo(0)
+    animator.start()
+    assertThat(AnimationHandler.getAnimationCount()).isEqualTo(1)
+
+    paparazzi.snapshot(view, timeNanos = 0L)
+    assertThat(log).containsExactly(
+      "onDraw text=",
+    )
+    log.clear()
+
+    paparazzi.snapshot(view, timeNanos = 2_000_000_000L)
```

Comment:
I could go either way on millis vs. nanos here. In general I prefer nanos 'cause the precision is potentially valuable (?) but I also don’t like how many annoying `0` characters I need to write here when specifying a time literally.

Reply:
Agreed, millis would be nicer here

Reply:
Done!